### PR TITLE
[2.2-Develop] Fix for issue #5726

### PR DESCRIPTION
**Pull request by PieterCappelle**
Was fixed in 2.0.8 branch. Never got into 2.1-develop and 2.2-develop. Cherry-pick from 9fc059bc5b5432cadfd2f1cfa3cc2516e607e068

https://github.com/magento/magento2/issues/5726

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by okorshenko** [comment]
Hi @PieterCappelle 
Could you please check failed unit tests?

**Comment by PieterCappelle** [comment]
Hi, one unittest failed because of execution limit, all other ones completed without errors. I think the problem is in travis and not in my code.

File: lib/internal/Magento/Framework/Mail/Template/TransportBuilder.php [hunk 1]
```diff
@@ -159,14 +162,26 @@ public function setReplyTo($email, $name = null)
     }
 
     /**
+     * Set scope
+     *
+     * @param int $scopeId
+     * @return $this
+     */
+    public function setScopeId($scopeId)
```
Comment (okorshenko) [inline lib/internal/Magento/Framework/Mail/Template/TransportBuilder.php:170]: According to Backward Compatibility Policy, we cannot add new method(s) to `@api` classes. http://devdocs.magento.com/guides/v2.0/contributor-guide/backward-compatible-development/#introduction-of-a-method-to-a-class-or-interface

**Comment by okorshenko** [comment]
Hi @PieterCappelle 
QA team reported GitHub issue #5726 is still reproducible on that branch. (same as on PR to 2.1-develop). Could you please double check

**Comment by PieterCappelle** [comment]
Can you tell me what could reproduced? Because I've edited multiple cases. Is it one specific case, because in my tests it's working like it should be.

File: lib/internal/Magento/Framework/Mail/Template/TransportBuilder.php (hunk 1 above)
Comment (PieterCappelle) [reply lib/internal/Magento/Framework/Mail/Template/TransportBuilder.php:170]: Okay, and how should I fix that? This function is necessary for the complete workflow of this PR.

**Comment by okorshenko** [comment]
Sure, I requested detailed information from the QA department. Should be ready soon

**Comment by okorshenko** [comment]
Hi @PieterCappelle 
Here is video provided by our QA
https://youtu.be/ng4bWnkc5-Q

**Comment by PieterCappelle** [comment]
So strange. Viewed the video and indeed same problem occurs. Will try this case. Will keep you updated.

**Comment by okorshenko** [comment]
Hi @PieterCappelle I'm closing this PR for now. Please, reopen it once ready

**Pull request closed without merging**